Files/ViewModels/ItemViewModel.cs

```diff
@@ -997,6 +997,70 @@ public async void RapidAddItemsToCollectionAsync(string path, string previousDir
                         }
                     }
                 }
+                if (AssociatedInstance.InstanceViewModel.FolderSettings.IsAdaptiveLayout)
+                {
+                    // TODO: Maybe first check here if desktop.ini is availabe
+                    // which we can get info about the folder from
+
+                    if (FilesAndFolders.Count == 0)
+                    {
+                        goto NoItemsLayoutSkip; // not evil
+                    }
+
+                    int imagesAndVideosCount = FilesAndFolders.Where((item) =>
+                        item.FileExtension == ".svg"
```

Comment:
Are we sure that `item.FileExtension` will always return a lowercase string? If not, we should compare with `.Equals()` instead.